sql/src/main/java/io/crate/planner/operators/LogicalPlanner.java

```diff
@@ -109,8 +109,8 @@ public LogicalPlan planSubSelect(SelectSymbol selectSymbol, PlannerContext plann
         if (selectSymbol.getResultType() == SINGLE_COLUMN_SINGLE_VALUE) {
             softLimit = fetchSize = 2;
```

Comment:
maybe a bit off-topic but why is this 2? can we document it?

Reply:
Are you referring to the fetch, limit size or both? Honestly, i am not sure why the fetch size is 2 and on what calculation it is based, couldn't find the reason yet.

Reply:
2 is because single value sub-queries need to have a validation that it only returns 1 row and not more.